src/schema/index.ts

```diff
@@ -50,6 +50,36 @@ const Schema = gql`
       user: ID
     ): Profile
   }
-`
+  type ratingSystem {
+    name:String!
+    grade:[Int]!
+    description:[String]!
+    percentage:[String]!
+  }
+  type Mutation {
+    createRatingSystem(name: String!, grade:[Int]!, description: [String]!, percentage: [String]!): ratingSystem!
```

Comment:
Remember to change the description type here to a string not an array of strings

Reply:
Well, I thought it is an array of strings since each elements in grade array will have its own description.